src/components/drawing/index.js

```diff
@@ -110,8 +110,10 @@ drawing.hideOutsideRangePoints = function(traceGroups, subplot) {
         var trace = d[0].trace;
         var xcalendar = trace.xcalendar;
         var ycalendar = trace.ycalendar;
-        var selector = trace.type === 'bar' ? '.bartext' :
+        var selector =
```

Comment:
Non-blocking (so feel free to ignore), but maybe we could clean this up by adding a field on `trace._module` (i.e. on trace `index.js` exports) e.g. 

```js
// in bar/index.js

module.exports = {
  // ...
  hideOutsideRangeSelector: '.bartext'
};
```

Reply:
So you didn't need to add `Drawing.setClipUrl` calls as in:

https://github.com/plotly/plotly.js/blob/aec946e8570c042d632352fc58eaf99276797bd5/src/traces/bar/plot.js#L170


in https://github.com/plotly/plotly.js/pull/3817/commits/04049f2ca3c440d4e4da429b1481f58cdd660daa to get this to work?

Reply:
We had that here: https://github.com/plotly/plotly.js/blob/e968da68abbc70c67a0b08f646d266a2e1754dd2/src/traces/funnel/plot.js#L74-L76

Reply:
Cool. Thanks!